diff -up e-smith-proxy-5.2.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/proxy.pm.transparent e-smith-proxy-5.2.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/proxy.pm
--- e-smith-proxy-5.2.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/proxy.pm.transparent	2010-09-26 10:18:56.000000000 -0600
+++ e-smith-proxy-5.2.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/proxy.pm	2010-09-26 10:22:17.000000000 -0600
@@ -117,7 +117,7 @@ sub show_smtp_proxy_status
 {
     my $self = shift;
     my $q = $self->{cgi};
-    my @smtp_proxy_settings = qw(enabled disabled blocked);
+    my @smtp_proxy_settings = qw(transparent disabled blocked);
     my $default = $self->get_smtp_proxy_status();
     my %labels = (
         enabled  => $self->localise('ENABLED'),
