diff -Nur -x '*.orig' -x '*.rej' e-smith-proxy-4.14.0/createlinks mezzanine_patched_e-smith-proxy-4.14.0/createlinks
--- e-smith-proxy-4.14.0/createlinks	2006-03-15 14:08:21.000000000 -0700
+++ mezzanine_patched_e-smith-proxy-4.14.0/createlinks	2007-01-25 15:11:20.000000000 -0700
@@ -26,6 +26,7 @@
 }
 
 templates2events("/etc/rc.d/init.d/masq", "proxy-update");
+templates2events("/etc/crontab", "proxy-update");
 safe_symlink("adjust", "root/etc/e-smith/events/proxy-update/services2adjust/masq");
 
 # Daemontools links.
