diff -Nur -x '*.orig' -x '*.rej' e-smith-proxy-4.14.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/proxy mezzanine_patched_e-smith-proxy-4.14.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/proxy
--- e-smith-proxy-4.14.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/proxy	2005-06-09 14:36:09.000000000 -0600
+++ mezzanine_patched_e-smith-proxy-4.14.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/proxy	2007-12-27 01:36:50.000000000 -0700
@@ -20,8 +20,8 @@
             proxy settings.
 
             The server includes a transparent proxy and cache for 
-            HTTP traffic. This is enabled by default, unless the 
-            server is in &quot;serveronly&quot; mode. 
+            HTTP traffic. This is enabled by default, but not enforced 
+            if the server is in &quot;serveronly&quot; mode. 
 
             If this server is acting as an e-mail server, connections 
             from local network clients to external SMTP servers 
