diff -Nur -x '*.orig' -x '*.rej' e-smith-proxy-4.13.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/proxy.pm mezzanine_patched_e-smith-proxy-4.13.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/proxy.pm
--- e-smith-proxy-4.13.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/proxy.pm	2005-07-14 00:43:19.868492112 -0600
+++ mezzanine_patched_e-smith-proxy-4.13.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/proxy.pm	2005-07-14 00:42:24.150162992 -0600
@@ -74,7 +74,7 @@
 sub get_smtp_proxy_status
 {
     my $self = shift;
-    return $self->{db}->get_prop('smtpfront-qmail', 'Proxy');
+    return $self->{db}->get_prop('smtpd', 'Proxy');
 }
 
 =head2 change_settings
@@ -93,9 +93,9 @@
 
     my $squid = $self->{db}->get('squid')
         or return $self->error('ERR_NO_SQUID_REC');
-    # smtpfront-qmail is allowed to not exist, as the 6040 might not be
+    # smtpd is allowed to not exist, as the relevant packages may not be
     # installed.
-    my $smtpd = $self->{db}->get('smtpfront-qmail') || undef;
+    my $smtpd = $self->{db}->get('smtpd') || undef;
 
     $squid->set_prop('status', $http_proxy_status);
     $smtpd->set_prop('Proxy', $smtp_proxy_status) if $smtpd;
