diff -Nur -x '*.orig' -x '*.rej' e-smith-proxy-4.14.0/root/etc/e-smith/templates/etc/dhcpd.conf/26wpad mezzanine_patched_e-smith-proxy-4.14.0/root/etc/e-smith/templates/etc/dhcpd.conf/26wpad
--- e-smith-proxy-4.14.0/root/etc/e-smith/templates/etc/dhcpd.conf/26wpad	2008-02-22 16:15:17.000000000 +0000
+++ mezzanine_patched_e-smith-proxy-4.14.0/root/etc/e-smith/templates/etc/dhcpd.conf/26wpad	2008-02-22 16:13:38.000000000 +0000
@@ -5,7 +5,7 @@
 
         {
 
-        $OUT .= 'option wpad-url    "http://wpad.yourproxy.you/proxy.pac";';
+        $OUT .= "    option wpad-url            \"http://wpad.$DomainName/wpad.dat\";";
 }
  else
         {
