diff -Nur -x '*.orig' -x '*.rej' e-smith-proxy-4.14.0/root/etc/e-smith/templates/etc/squid/squid.conf/50-transproxy mezzanine_patched_e-smith-proxy-4.14.0/root/etc/e-smith/templates/etc/squid/squid.conf/50-transproxy
--- e-smith-proxy-4.14.0/root/etc/e-smith/templates/etc/squid/squid.conf/50-transproxy	2002-04-19 12:45:31.000000000 -0400
+++ mezzanine_patched_e-smith-proxy-4.14.0/root/etc/e-smith/templates/etc/squid/squid.conf/50-transproxy	2008-08-15 14:20:04.000000000 -0400
@@ -1,5 +1,5 @@
 {
-    my $transproxy = db_get_prop($confref, 'squid', 'Transparent') || "yes";
+    my $transproxy = $squid{Transparent} || "yes";
     $OUT = '';
     return unless ($transproxy eq "yes");
 
