diff -Nur -x '*.orig' -x '*.rej' e-smith-proxy-4.13.2/createlinks mezzanine_patched_e-smith-proxy-4.13.2/createlinks
--- e-smith-proxy-4.13.2/createlinks	2005-03-09 14:32:25.000000000 +1100
+++ mezzanine_patched_e-smith-proxy-4.13.2/createlinks	2006-03-13 17:21:21.994456022 +1100
@@ -8,6 +8,7 @@
     bootstrap-console-save
     network-create
     network-delete
+    proxy-update
 ))
 {
     templates2events("/etc/httpd/conf/proxy/proxy.pac", $event);
